### DISCUSSION TITLE
Dialog focus on close

### DISCUSSION THREAD

**Discussion by Emiliano-Bucci** [radix-ui/primitives#2452] [Help]
Hi! Is there a way to remove trap focus when prop `open` passed to `Dialog.Root` is **false**? The use case is that I'm handling the dialog and animating with react spring and the problem is that if I close the dialog then I have to wait till the trap focus is removed (which is not instant) so I can't open/close (toggle) dialogs. Thanks!

**Comment by flipvh** [comment]
Hi Emiliano, did you find a solution? I am also looking into a way to disable the focus trap entirely: https://github.com/radix-ui/primitives/discussions/2533
